modules/siddhi-core/src/main/java/org/wso2/siddhi/core/table/InMemoryEventTable.java

```diff
@@ -41,17 +41,14 @@
 import org.wso2.siddhi.query.api.expression.Expression;
 import org.wso2.siddhi.query.api.util.AnnotationHelper;
 
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
+import java.util.*;
 
 public class InMemoryEventTable implements EventTable, Snapshotable {
 
     private final TableDefinition tableDefinition;
     private final ExecutionPlanContext executionPlanContext;
-    private LinkedList<StreamEvent> list = new LinkedList<StreamEvent>();
-    private TreeMap<Object, StreamEvent> treeMap = new TreeMap<Object, StreamEvent>();
+    private List<StreamEvent> list = new LinkedList<StreamEvent>();
```

Comment:
It's better to add proper names for variables,rather generic terms as list,treemap